scripts/lib/update_acme_tests.py

```diff
@@ -53,7 +53,8 @@
                              "SMS.T42_T42.S",
                              "PRE.f19_f19.ADESP",
                              "PRE.f19_f19.ADESP_TEST",
-                             "MCC_P1.f19_g16_rx1.A")
+                             "MCC_P1.f19_g16_rx1.A",
+                             "LDSTA.ne4np4_oQU240.A")
```

Comment:
I don't think that this grid is defined for cesm

Reply:
Okay. What's the smallest grid e3sm and cesm support? Or does it affect A case runtimes and/or output file sizes?

Reply:
f19_g16_rx1 is typical for testing

Reply:
I think f19 is much higher res then ne4.  Try f45_g37_rx1